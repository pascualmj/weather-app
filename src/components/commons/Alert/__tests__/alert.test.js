import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Alert from "../Alert";

describe("Component: Alert", () => {
  test("Should match snapshot.", () => {
    const { asFragment } = render(<Alert text="a" type="success" />);

    expect(asFragment()).toMatchSnapshot();
  });

  test("Should render text content.", () => {
    const { getByText } = render(<Alert text="b" />);

    expect(getByText(/b/i)).not.toBeNull();
  });
});
