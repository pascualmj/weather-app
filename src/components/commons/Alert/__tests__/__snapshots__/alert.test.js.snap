// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Alert Should match snapshot. 1`] = `
<DocumentFragment>
  <div
    class="alert success "
  >
    <p
      class="text color-absence size-small weight-regular "
    >
      a
    </p>
  </div>
</DocumentFragment>
`;
